packages/playground/app/_layouts/empty.html

```diff
@@ -0,0 +1,7 @@
+<style>
+    .page-header,
+    .side-bar {
+        display: none !important;
```

Comment:
!important most probably not needed, since you control all the HTML and CSS

Reply:
Yes, you are right that it is not needed. It is there to make obvious the purpose of this template.